### ISSUE TITLE
to_array + spacy.en.attrs.TAG = zero array

### ISSUE THREAD

**Issue by flor385** [honnibal/spaCy#43]
I've had some problems converting certain token attributes to numpy arrays. From what I understand, extracting TAG attributes should work as well as extracting POS attributes, but it doesn't. Following is minimal demonstration.

Also, it would be great if dependency types (and perhaps even token.head indices) could be extracted using the same API.

```
import spacy.en
import numpy as np

nlp = spacy.en.English()

toks = nlp(u"This is a simple sentence.", True, True)

print "Extracting google POS"
print np.array(toks.to_array([spacy.en.attrs.POS]))

print "Extracting detailed TAG doesn't"
print np.array(toks.to_array([spacy.en.attrs.TAG]))

print "Even though the detailed TAG is detected"
print [t.tag for t in toks]
```

**Comment by syllog1sm** [comment]
Thanks, will fix.

**Comment by honnibal** [comment]
Fixed in 0.80
